src/Servers/IIS/AspNetCoreModuleV2/IISLib/IISLib.vcxproj

```diff
@@ -40,22 +40,20 @@
     <RootNamespace>IISLib</RootNamespace>
     <ProjectName>IISLib</ProjectName>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
-    <PlatformToolsetVersion Condition=" '$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolsetVersion>
-    <PlatformToolsetVersion Condition=" '$(PlatformToolsetVersion)' == '' ">v142</PlatformToolsetVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">
-    <VCToolsVersion />
+    <VCToolsVersion>14.29.30133</VCToolsVersion>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
-    <UseDebugLibraries>true</UseDebugLibraries>
+    <UseDebugLibraries>false</UseDebugLibraries>
```

Comment:
Tracked somewhere so we can revert eventually?

Reply:
Depends on whether we can convince runtime to produce a debug version of libnethost, but yeah good idea I'll file an issue tracking this

Reply:
Added it to the meta issue https://github.com/dotnet/aspnetcore/issues/39281